WooCommerce/Classes/ViewRelated/Products/Media/ProductImagesViewController.swift

```diff
@@ -237,7 +272,7 @@ extension ProductImagesViewController {
     }
 
     private func hasOutstandingChanges() -> Bool {
-        return hasDeletedAnyImages
+        return hasDeletedAnyImages || hasMovedAnyImages
```

Comment:
❓  By this, we're considering any movement in the list an outstanding change, even the case when the list ends up with the old order (e.g: when we move an item to a new position and then move it back). Do you think it's worth it to keep track of the order too?

Reply:
Good point! 🤔 TBH, we should keep track the order for a better UX, but I can't find a simple way to do it...I mean, `ProductImageStatus` is neither `Equatable` nor `Identifiable`, so we should analyze all elements to validate that nothing changed. Is it worth it? 🤔 

WDYT? 😅